Products/SQLAlchemyDA/da.py

```diff
@@ -300,7 +300,8 @@ def getInfo(self):
         if wrapper is not None:
             d = self.sa_zope_wrapper().__dict__.copy()
             d['DSN'] = self.sa_zope_wrapper().dsn
-            for k in d.keys()[:]:
+            dd = d.copy()
+            for k in dd.keys():
                 if k.startswith('_'):
```

Comment:
what about..
`for k in list(d.keys())`

That way you do not need to copy the complete dictionary.